lua/fishing_mod/cl_init.lua

```diff
@@ -1,7 +1,5 @@
 fishingmod = fishingmod or {}
 
-local color_white = color_white or Color(255, 255, 255, 255)

```

Comment:
But why??

Reply:
Micro-optimization

Reply:
That is not micro-optimization, you are doing the opposite here. 
Grabbing from a local is faster than global lookup for color_white

Reply:
It's micro-optimization compared to caching. Here you create a variable for nothing.

Reply:
It is used 7 other times thus isn't "for nothing"